packages/Textarea/stories/Textarea.stories.js

```diff
@@ -40,10 +32,3 @@ variations.story = {
   name: "Variations",
   parameters: variationsStoryParameters,
 };
-
-// storiesOf(`${storyName}/Examples`, module)
-//   .add("Uncontrolled", Uncontrolled)
-//   .add("Sizes", () => <Sizes />);
-
-// storiesOf(`${storyName}/Backyard/Tests`, module)
-//   .add("Screener", () => <ScreenerStory />)
```

Comment:
My apologies, I seem to have forgotten that these stories had already been refactored to CSF previously – most of the changes in this PR are not actually needed, and we can just close the JIRA tickets.  

But it would be nice to clear out these unnecessary comments. I'm not sure if you'd prefer to revert the other changes in this PR and merge this small cleanup – or just close this PR, and sneak this small cleanup to `Textarea.stories.js` into your other PR?